force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -134,7 +181,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the date is in the past, False otherwise.
      */
     public static Boolean isDateInPast(Date dt) {
+        if(dt == null)
```

Comment:
For boolean, you want to be careful about returning null as a response. Whatever calls your method would need to deal with it.